<launch>
    <rosparam command="load" file="$(find slam)/config/plane_segmentation.yaml"/>
    
    <include file="$(find slam)/launch/image2pointcloud.launch"/>
    
    <node pkg="slam" type="plane_segmentation.py" name="plane_segmentation" output="screen"/>
</launch>